packages/apps/docs/src/pages/pact/beginner/hello-world.md

```diff
@@ -106,11 +106,10 @@ our GitHub.
 ### Create a Keyset
 
 As you can see, the “Hello World” smart contract starts by defining and reading
```

Comment:
Above "## Hello World Overview" section hyperlinks are not proper. (pact.kadena.io)[http://pact.kadena.io]. it should be [pact.kadena.io](http://pact.kadena.io)

it's not part of this PR. Since you're already making changes thought of adding it here